ksa_compliance/output_models/e_invoice_output_model.py

```diff
@@ -1135,4 +1136,5 @@ def get_e_invoice_details(self, invoice_type: str):
         self.result["invoice"]["total_taxes_and_charges_percent"] = sum(
             it.rate for it in self.sales_invoice_doc.get("taxes", []))
         self.result["invoice"]["item_lines"] = item_lines
+        self.result["invoice"]["actual_line_extension_amount"] = sum(it["amount"] for it in item_lines)
```

Comment:
Is there another line extension amount value? Why do we need to call this `actual`?